androidClient/app/src/main/java/com/csatimes/dojma/activities/AboutUsActivity.java

```diff
@@ -29,20 +32,44 @@
 
     public static String ABOUT_US_FACEBOOK_URL = "https://www.facebook.com/MACBITSGoa";
     public static String ABOUT_US_FACEBOOK_PAGE_ID = "MACBITSGoa";
+    public static String ABOUT_US_GITHUB_URL = "https://github.com/MobileApplicationsClub";
```

Comment:
I meant all of the urls to be taken from firebase. Also, take urls for the icons from firebase.